WordPress/src/main/java/org/wordpress/android/ui/reader/repository/ReaderRepositoryEvents.kt

```diff
@@ -39,5 +60,13 @@ sealed class ReaderDiscoverCommunication {
     sealed class Error(open val task: DiscoverTasks) : ReaderDiscoverCommunication() {
         data class NetworkUnavailable(override val task: DiscoverTasks) : Error(task)
         data class RemoteRequestFailure(override val task: DiscoverTasks) : Error(task)
+        data class ServiceNotStarted(override val task: DiscoverTasks) : Error(task)
     }
 }
+
+fun ReaderDiscoverCommunication.isRequestMore(): Boolean =
```

Comment:
This is works as well. However, we might want to consider refactoring the ReaderDiscoverCommunication and use it.task == REQUEST_MORE. Wdyt?

```
sealed class ReaderDiscoverCommunication {
    abstract val task: DiscoverTasks
    data class Started(override val task: DiscoverTasks) : ReaderDiscoverCommunication()
    data class Success(override val task: DiscoverTasks) : ReaderDiscoverCommunication()
    sealed class Error : ReaderDiscoverCommunication() {
        data class NetworkUnavailable(override val task: DiscoverTasks) : Error()
        data class RemoteRequestFailure(override val task: DiscoverTasks) : Error()
    }
}
```

Reply:
@malinajirka - Yes!! I thought we weren't touching this until our refactoring chat, but I'll consider your comment the go-ahead, so I'm implementing now. :) Thanks. In commit 39a833a